src/ui/React/ImportSave/ImportSave.tsx

```diff
@@ -1,38 +1,43 @@
 import React, { useEffect, useState } from "react";
 
-import {
-  Paper,
-  Table,
-  TableHead,
-  TableRow,
-  TableBody,
-  TableContainer,
-  TableCell,
-  Typography,
-  Tooltip,
-  Box,
-  Button,
-  ButtonGroup,
-} from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import Collapse from "@mui/material/Collapse";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Table from "@mui/material/Table";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableBody from "@mui/material/TableBody";
+import TableContainer from "@mui/material/TableContainer";
+import TableCell from "@mui/material/TableCell";
+import Tooltip from "@mui/material/Tooltip";
+import Typography from "@mui/material/Typography";
```

Comment:
I think we decided there was no reason we were importing from individual files, and we can just import mui stuff all from the main mui/material. But it's not really a big deal so I'm not going to kick it back for that.

Reply:
I actually could not remember what we decided xD
I've just checked again and d0sboots mentioned that our imports are mostly consistent, most of them are like in this PR (import from subdirectories).
I also prefer this kind of import for mui. In some cases, you also want to import some types (e.g. `SelectProps`) and it's better to have imports grouped in this case.